src/anomalib/models/efficientad/lightning_model.py

```diff
@@ -170,16 +170,43 @@ def map_norm_quantiles(self, dataloader: DataLoader) -> dict[str, Tensor]:
                     map_ae = output["map_ae"]
                     maps_st.append(map_st)
                     maps_ae.append(map_ae)
-        maps_st = torch.cat(maps_st)
-        maps_ae = torch.cat(maps_ae)
-        qa_st = torch.quantile(maps_st, q=0.9).to(self.device)
-        qb_st = torch.quantile(maps_st, q=0.995).to(self.device)
-        qa_ae = torch.quantile(maps_ae, q=0.9).to(self.device)
-        qb_ae = torch.quantile(maps_ae, q=0.995).to(self.device)
+
+        qa_st, qb_st = self.get_quantiles_of_maps(maps_st)
```

Comment:
I believe this line throws an error unless changed to self._get_quantiles_of_maps()
```suggestion
        qa_st, qb_st = self._get_quantiles_of_maps(maps_st)
```

Reply:
same with line 175